more/pixels/pixelMintKeeper.js

```diff
@@ -77,36 +76,27 @@ export function makePixelMintKeeper(assay) {
     // that it is currently in. Destroy is outside of an assetKeeper
     // because it could affect purses *or* payments
     destroy(amount) {
-      // amount must only contain a pixelList of length 1 for now
       const pixelList = assay.quantity(amount);
-      insist(
-        pixelList.length === 1,
-      )`amount must contain a pixelList of length 1 for now`;
+      pixelList.forEach(pixel => {
+        const strPixel = getString(pixel);
+        if (pixelToAsset.has(strPixel)) {
+          const asset = pixelToAsset.get(strPixel);
+          amount = assay.coerce(amount);
 
-      const pixel = pixelList[0];
-      const strPixel = getString(pixel);
-      insist(
-        pixelToAsset.has(strPixel),
-      )`pixel ${strPixel} could not be found to be destroyed`;
-      const asset = pixelToAsset.get(strPixel);
-      // amount is guaranteed to be there
-      amount = assay.coerce(amount);
+          const keeper = getKeeper(asset);
+          const originalAmount = keeper.getAmount(asset);
+          const newAmount = assay.without(originalAmount, amount);
```

Comment:
The two amounts on the right are very different things. I think I'd try to distinguish between purseBalance and removePixelAmount. It might be okay to leave the second arg as amount if it started as allegedAmount or requestedAmount.

Reply:
I think this code also wasn't changed in this PR, apart from putting it in an `if` block, but I would agree with you that `originalAmount` and `amount` are not very well distinguished. I can make the change you suggested in this PR.